pkg/vault/util.go

```diff
@@ -0,0 +1,25 @@
+package vault
+
+import (
+	"fmt"
+	"os"
+)
+
+// InitVault TODO
+func InitVault(vaultType string) (VaultType, error) {
```

Comment:
Does this function get used? seems that `pkg/vault/config.go`'s `NewConfig()` does the job